test/e2e/pivoting.go

```diff
@@ -50,6 +50,7 @@ type PivotingInput struct {
 	ClusterctlConfigPath  string
 }
 
+// Pivoting implements a test that verifies moving management resources (CRs, BMO, Ironic) to a target cluster after initializing it with Provider components.
```

Comment:
Nitpicking a bit, but verifies what exactly?

Reply:
it verifies "moving management resources" the movement of crs to target cluster

Reply:
"Pivoting implements a test that verifies successful moving of management resources (CRs, BMO, Ironic) to a target cluster after initializing it with Provider components." or something along those lines is what I was after.

Reply:
thanks